public/assets/sass/components/_header.scss

```diff
@@ -0,0 +1,5 @@
+// .header {
```

Comment:
I think this file belongs to the layout directory not to the components as it is not a component I think. It is already added to the layout directory. So, if this is not needed any more, please remove it and its reference in the `main.scss` file too.